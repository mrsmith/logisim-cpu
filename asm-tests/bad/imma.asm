# no imm
imma

# not a number
imma xxx

# too large
imma 256

# too small
imma -129
