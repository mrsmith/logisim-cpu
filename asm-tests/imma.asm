imma 42
hlt
