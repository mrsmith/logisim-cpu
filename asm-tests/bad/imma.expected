imma.asm:2: warn: bad imma instruction `imma`
imma.asm:5: warn: bad imma instruction `imma xxx`
imma.asm:8: warn: bad imma imm `256`
imma.asm:11: warn: bad imma imm `-129`
